llvm/lib/Support/CommandLine.cpp

```diff
@@ -2351,6 +2351,20 @@ sortSubCommands(const SmallPtrSetImpl<SubCommand *> &SubMap,
 
 namespace {
 
+/// HelpCategory defines various category groups for dpct options
+enum class HelpCategory {
```

Comment:
this file is not only for DPCT, so guard the code change with macro: SYCLomatic_CUSTOMIZATION